<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkinnedSettings</class>
 <widget class="QWidget" name="SkinnedSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Skins</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_66">
       <item row="0" column="0" colspan="4">
        <widget class="QListWidget" name="listWidget">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="flow">
          <enum>QListView::TopToBottom</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="skinInstallButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Add...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="skinReloadButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>View</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>View</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QCheckBox" name="hideOnCloseCheckBox">
            <property name="text">
             <string>Hide on close</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="hiddenCheckBox">
            <property name="text">
             <string>Start hidden</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="skinCursorsCheckBox">
            <property name="text">
             <string>Use skin cursors</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="plGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Fonts</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Player:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="mainFontLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>???</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="mainFontButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Playlist:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="plFontLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>???</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="plFontButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QCheckBox" name="useBitmapCheckBox">
            <property name="text">
             <string>Use bitmap font if available</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Transparency</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Main window</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="mwTransparencySlider">
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="mwTransparencyLabel">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Equalizer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="eqTransparencySlider">
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="eqTransparencyLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Playlist</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="plTransparencySlider">
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="plTransparencyLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Miscellaneous</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Song Display</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="protocolCheckBox">
            <property name="text">
             <string>Show protocol</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="numbersCheckBox">
            <property name="text">
             <string>Show song numbers</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="alignCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Align song numbers</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="anchorCheckBox">
            <property name="text">
             <string>Show anchor</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="playlistsCheckBox">
            <property name="text">
             <string>Show playlists</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="popupCheckBox">
            <property name="text">
             <string>Show popup information</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="popupTemplateButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Edit template</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Playlist separator:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="plSeplineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>URL Dialog</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="clipboardCheckBox">
            <property name="text">
             <string>Auto-paste URL from clipboard</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>144</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mwTransparencySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mwTransparencyLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eqTransparencySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>eqTransparencyLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plTransparencySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>plTransparencyLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>popupCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>popupTemplateButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numbersCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>alignCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playlistsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>plSeplineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
